### PR TITLE
feat: Store firebase device id

### PR DESCRIPTION
**Pull request by beaapascual**
Added attributes to update user to store fcm data

### CODE REVIEW

File: request/user/index.js [hunk 1]
```diff
@@ -41,7 +41,10 @@ module.exports.update = {
 			isBlocked: {type: ['boolean', 'null']},
 			passwordReset: {type: ['boolean', 'null']},
 			description: {type: 'string', minLength: 1},
-			pictures: file.list
+			pictures: file.list,
+			platform: {type: 'string', minLength: 1},
```
Comment (bangretraced) [inline request/user/index.js]: should be `{type: ['string', 'null'], minLength: 1}` for all 3 additional fields

File: request/user/index.js [hunk 2]
```diff
@@ -41,7 +41,10 @@ module.exports.update = {
 			isBlocked: {type: ['boolean', 'null']},
 			passwordReset: {type: ['boolean', 'null']},
 			description: {type: 'string', minLength: 1},
-			pictures: file.list
+			pictures: file.list,
+			platform: {type: ['string', 'null'], minLength: 1},
```
Comment (bangretraced) [inline request/user/index.js]: sorry, i was wrong. 
These 3 additional fields should not be here. You need to create new schema just for endpoint `/me/device-token`. And those fields cannot be null in schema

Comment (bangretraced) [reply request/user/index.js]: create new schema 
```
module.exports.updateDeviceToken {
body: {
		type: 'object',
		description: 'Users Device token',
		required: ['platform', 'appType', 'deviceId'],
		additionalProperties: false,
              properties: {}
}
```

File: request/user/index.js [hunk 3]
```diff
@@ -67,6 +67,20 @@ module.exports.profile = {
...
 }
 
+module.exports.updateDeviceToken  = {
+	body: {
+		type: 'object',
+		description: 'Users Device token',
+		required: ['platform', 'appType', 'deviceId'],
+		additionalProperties: false,
+        properties: {
+			platform: {type: ['string', 'null'], minLength: 1},
```
Comment (bangretraced) [inline request/user/index.js]: not null here `{type: 'string', minLength: 1}`

Comment (bangretraced) [reply request/user/index.js]: Please mark is as Resolved if it was changed already

File: request/user/index.js [hunk 4]
```diff
@@ -67,6 +67,20 @@ module.exports.profile = {
 	}
 }
 
+module.exports.updateDeviceToken  = {
```
Comment (bangretraced) [inline request/user/index.js]: it should be `createDeviceToken`

Comment (bangretraced) [reply request/user/index.js]: ignore this

File: request/user/index.js [hunk 5]
```diff
@@ -67,6 +67,20 @@ module.exports.profile = {
 	}
 }
 
+module.exports.createDeviceToken  = {
+	body: {
+		type: 'object',
+		description: 'Users Device token',
+		required: ['platform', 'appType', 'id'],
```
Comment (bangretraced) [inline request/user/index.js]: no, here dont change deviceId. keeps as before

Comment (beaapascual) [reply request/user/index.js]: haha got it :)

File: request/user/index.js [hunk 6]
```diff
@@ -67,6 +67,20 @@ module.exports.profile = {
...
 
+module.exports.createDeviceToken  = {
+	body: {
+		type: 'object',
+		description: 'Users Device token',
+		required: ['platform', 'appType', 'deviceId'],
+		additionalProperties: false,
+        properties: {
+			platform: {type: 'string', minLength: 1},
+			appType: {type: 'string', minLength: 1},
```
Comment (bangretraced) [inline request/user/index.js]: create new constants for `appType`. Add in `lib/constatns.js`
```
module.exports.APPTYPES_ENUM = {
	CONSUMER: 'CONSUMER',
	BUSINESS: 'BUSINESS'
}
module.exports.APPTYPES = Object.keys(exports.APPTYPES_ENUM)
```
and change to `appType: {enum: constants.APPTYPES}`

**Pull request merged**